lib/python/Plugins/Extensions/FileCommander/plugin.py

```diff
@@ -325,7 +325,8 @@ def __init__(self, session, pathLeft="", pathRight="", leftActive=None):
 			"right": (self.keyGoRightColumn, _("Switch to the right column")),
 			"down": (self.keyGoLineDown, _("Move down a line")),
 			"pageDown": (self.keyGoPageDown, _("Move down a screen")),
-			"bottom": (self.keyGoBottom, _("Move to last line / screen"))
+			"bottom": (self.keyGoBottom, _("Move to last line / screen")),
+			"startTeletext": (self.keyViewEdit, _("View or edit file (if size < 1MB)"))
```

Comment:
There is no protection.
The key is always working and this is not good.

Reply:
I want this key always working but

I'm sorry. Feel free to improve. Maybe i can see and learn.